src/react-queries/index.ts

```diff
@@ -152,12 +261,11 @@ const thornode = createQueryKeys('thornode', {
     }
   },
   inboundAddress: (assetId: AssetId | undefined) => {
-    if (!assetId) throw new Error('assetId is required')
-
     return {
       staleTime: 60_000, // 60 seconds to handle pools going to/from live/halt states
       queryKey: ['thorchainInboundAddress', assetId],
       queryFn: async () => {
+        if (!assetId) throw new Error('assetId is required')
```

Comment:
That was a bigly mistake on my part having this one outside of the `queryFn` - runs of `useQuery(inboundAddress(assetId))`  with `undefined` `assetId` would crash the app